demo/api/src/products/generated/dto/product-category.input.ts

```diff
@@ -16,6 +16,12 @@ export class ProductCategoryInput {
     @IsSlug()
     @Field()
     slug: string;
+
+    @IsNullable()
+    @Min(1)
+    @IsInt()
+    @Field(() => Int, { nullable: true })
+    position?: number;
```

Comment:
This can also be undefined, so you either need `@IsUndefinable()` or use `@IsOptional()` instead for both decorators

Reply:
But should this even be nullable (= set position to null)? Shouldn't it be undefinable only?

Reply:
IsUndefinable is never used. I'll go with IsOptional. I used IsNullable because crud-generator already checked for prop.nullable (depending on the @Field-Decorator) adding IsNullable/IsNotEmpty.